broll/src/frame_decoder.cpp

```diff
@@ -14,6 +14,8 @@
 
 extern "C" {
 #include <libavcodec/avcodec.h>
+#include <libavcodec/bsf.h>
```

Comment:
What symbols are included from this?

Reply:
In 4.x almost nothing, but in 6.x `AVBitstreamFilter` is defined there instead of in the toplevel `avcodec.h`, but the header exists in both `libavcodec` versions so it's safe to include